forem/forem
# [15 min fix] Initialize follow button on podcast page

## What type of PR is this? (check all applicable)


## Description

The podcast page was missed in the refactor of follow button initialization - this PR makes sure the correct pack runs on this page and the follow button behaviour is reinstated.

## Related Tickets & Documents

Fixes #14553

## QA Instructions, Screenshots, Recordings

When logged in:
- Visit a podcast page e.g. '/codenewbie'
- Check the Follow button in the header has the correct 'Follow' text
- Check that when you click the button it changes to 'Following' and the 'follows' network request is triggered in the dev tools
- Refresh the page to make sure the Following state is persisted properly
- Repeat the process and check it turns back to 'Follow'

When logged out:
- Visit a podcast page e.g. '/codenewbie'
- Check the Follow button in the header has the correct 'Follow' text
- Check that when you click the button, the login modal appears


https://user-images.githubusercontent.com/20773163/130211955-41a7d209-92ee-408d-9c3a-8249a96b3aa1.mp4


### UI accessibility concerns?

N/A

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
